core/src/components/textarea/test/basic/textarea.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('textarea: basic', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/textarea/test/basic`);
+
+    await page.setIonViewport();
+
+    expect(await page.screenshot()).toMatchSnapshot(`textarea-diff-${page.getSnapshotSettings()}.png`);
+  });
+
+  test.describe('with floating labels', () => {
```

Comment:
This looks to be a duplicate test of the `#dynamicValue` textarea in the above screenshot.